### PR TITLE
Avoid copying the array of TurboFilters for each call of any method of the loggers

### PR DESCRIPTION
**Pull request by A-Imal**
https://hivemq.kanbanize.com/ctrl_board/42/cards/17304/details/

### CODE REVIEW

File: src/main/java/com/hivemq/logging/modifier/XodusEnvironmentImplLogLevelModifier.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import ch.qos.logback.core.spi.FilterReply;
+import com.hivemq.extension.sdk.api.annotations.NotNull;
+import com.hivemq.extension.sdk.api.annotations.Nullable;
+import jetbrains.exodus.ExodusException;
+import jetbrains.exodus.env.EnvironmentImpl;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Marker;
+
+/**
+ * This Log Level Modifier is used to modify the log level of Xodus logs.
```
Comment (Remit) [inline src/main/java/com/hivemq/logging/modifier/XodusEnvironmentImplLogLevelModifier.java:30]: this comment is useless, pls remove

File: src/main/java/com/hivemq/logging/modifier/XodusFileDataWriterLogLevelModifier.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        if (level.isGreaterOrEqual(Level.WARN)) {
+            if (logger.equals(fileDataWriterLogger)) {
+                if (format.startsWith("Can't open directory channel. Log directory fsync won't be performed.")) {
+                    if (first.getAndSet(false)) {
+                        logger.debug("Can't open directory channel. Log directory fsync won't be performed.");
+                    }
+                    return FilterReply.DENY;
+                }
+            }
+
```
Comment (Remit) [inline src/main/java/com/hivemq/logging/modifier/XodusFileDataWriterLogLevelModifier.java:58]:
```suggestion
```

**Review by Remit — APPROVED** [review]

File: src/main/java/com/hivemq/logging/modifier/XodusEnvironmentImplLogLevelModifier.java (hunk 1 above)
Comment (A-Imal) [reply src/main/java/com/hivemq/logging/modifier/XodusEnvironmentImplLogLevelModifier.java]: done

File: src/main/java/com/hivemq/logging/modifier/XodusFileDataWriterLogLevelModifier.java (hunk 2 above)
Comment (A-Imal) [reply src/main/java/com/hivemq/logging/modifier/XodusFileDataWriterLogLevelModifier.java]: done

**Pull request merged**